common/setups/rasr/util.py

```diff
@@ -429,47 +461,235 @@ def __init__(
         self.feature_scorer = feature_scorer
 
 
+class RecognitionArgs:
+    def __init__(self, name, recognition_args):
+        self.name = name
+        self.recognition_args = recognition_args
+
+
+class OutputArgs:
+    def __init__(self, name):
+        self.name = name
+        self.corpus_key_type_pair = {}
+
+    def add_corpus_key_type_pair(self, corpus_key, corpus_type):
+        self.corpus_key_type_pair[corpus_key] = corpus_type
```

Comment:
optional cosmetic change:
```suggestion
        self.corpus_type_mapping = {}

    def define_corpus_type(self, corpus_key, corpus_type):
        self.corpus_type_mapping[corpus_key] = corpus_type
```